hoot-core-test/src/test/cpp/hoot/core/io/OsmApiWriterTest.cpp

```diff
@@ -92,7 +100,13 @@ class OsmApiWriterTest : public HootTestFixture
   void runCapabilitesTest()
   {
     QUrl osm;
+#ifdef RUN_LOCAL_TESTS
+    osm.setUrl(LOCAL_API_URL.arg(PORT_CAPABILITIES));
+    CapabilitiesTestServer server(PORT_CAPABILITIES);
+    server.start();
```

Comment:
The `RUN_LOCAL_TESTS` version starts a dummy test server that, in the case of the `runCapabilitiesTest` only responds with an OSM API capabilities response so that it can be run consistently and not rely on access to the actual OSM API.